easybuild/easyconfigs/s/summovie/summovie-1.0.2.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PackedBinary'
+
+name = 'summovie'
+version = '1.0.2'
+
+homepage = 'http://grigoriefflab.janelia.org/unblur'
+description = """Movie frame sums can be calculated using summovie,
+ which uses the alignment results from a prior run of Unblur."""
+
+toolchain = SYSTEM
+
+sources = [{
+    'source_urls': ['https://grigoriefflab.umassmed.edu/system/'],
+    'filename': '%(name)s_%(version)s.tar.gz',
+    'download_filename': 'tdf?path=%(name)s_%(version)s.tar.gz&file=1&type=node&id=4902',
+}]
+
+checksums = ['e64e25e588ac9951633fe626d8728d4d01debee205c1205dbf1c1b6ed27f0ce4']
+
+install_cmd = "cp -r %(namelower)s_%(version)s/bin %(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/sum_movie_openmp_7_17_15.exe'],
```

Comment:
Looks like you can compile this from source pretty easily (from https://grigoriefflab.umassmed.edu/unblur_summovie), that would be much more portable.

Reply:
Gave it a try, and it doesn't seem to compile easily :) First, the generated Makefile lists sources in the wrong order, wich leads to errors like: 
```
../core/orientations_and_rotations.f90:3:9:

    3 |     use Units
      |         1
Fatal Error: Cannot open module file ‘units.mod’ for reading at (1): No such file or directory
```
After patching it and fixing the order, the build still fails with
```
../core/sum_movie_routines.f90:76:76:

   76 |         real                    ::  x_drifts(size(x_shifts)), y_drifts(size(y_shifts))
      |                                                                            1
Error: Dummy argument ‘y_shifts’ at (1) cannot be OPTIONAL
```
I'm not a Fortran guru, but this sort of makes sense. Here is a source code from summovie:
```
        real,                           optional,   intent(in)      ::  x_shifts(:)             !<  X shifts (pixels)
        real,                           optional,   intent(in)      ::  y_shifts(:)             !<  Y shifts (pixels)
        ...
        real                    ::  x_drifts(size(x_shifts)), y_drifts(size(y_shifts))
```
So,  basically `x_drifts` and `y_drifts` are undefined, optional, but are used in the code. The issue sounds similar to this one: https://github.com/j3-fortran/fortran_proposals/issues/22 (still open).  So, I assume, allocation of these arrays should be enclosed by `if (present(x_shifts))` and `if (present(y_shifts))` statements. But again, not Fortran guru, so maybe it's enough to add a compiler flag to FFALGS.